### PR TITLE
Add TableSet component with dummy data to Models component

### PR DESCRIPTION
**Pull request by barrymcgee**
## Done

Add TableSet component with dummy data to Models component

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/

## Details

Fixes https://github.com/canonical-web-and-design/juju-squad/issues/811,
Fixes https://github.com/canonical-web-and-design/juju-squad/issues/810

### CODE REVIEW

**Comment by barrymcgee** [comment]
@hatched Why does [this for loop](https://github.com/canonical-web-and-design/jaas-dashboard/pull/24/files#diff-b4034ab556e902e12fdef5a8ad0e005eR112) only render one table despite there being two in the set?

File: src/App.js [hunk 1]
```diff
@@ -5,11 +5,18 @@ import ErrorBoundary from "./components/ErrorBoundary/ErrorBoundary";
...
 
+import TableSet from "./components/TableSet/TableSet";
+
 import "./scss/_layout.scss";
 
 // All following components are placeholders and will be replaced with imports.
 function Models() {
-  return <h2>Models</h2>;
+  return (
+    <>
```
Comment (hatched) [inline src/App.js]: Ahh what is this sorcery! 😆

Comment (barrymcgee) [reply src/App.js]: Screenshot 2019-08-15 at 19 29 02

Comment (hatched) [reply src/App.js]: Bahahahaha

**Pull request merged**